app/controllers/lessons_controller.rb

```diff
@@ -100,4 +99,10 @@ def set_course
     @course = Course.friendly.find(params[:course_id])
   end
 
+
+  def dl_subdomain
```

Comment:
@coreyd303 Maybe rename this `auth_other_subdomains`. I think it might be a bit confusing since the methods in other controllers are named the same but do different things. Then we can make this more clear and just have to change the method's name here and in the before filter.